cg/meta/backup/backup.py

```diff
@@ -158,9 +159,18 @@ def unlink_files(
     @staticmethod
     def create_rta_complete(decrypted_flow_cell: Path, run_dir: Path):
         """Create an RTAComplete.txt file in the flow cell run directory."""
-        (
-            run_dir / Path(decrypted_flow_cell.stem).stem / DemultiplexingDirsAndFiles.RTACOMPLETE
-        ).touch()
+        rta_complete_file = Path(
+            run_dir, decrypted_flow_cell.stem, DemultiplexingDirsAndFiles.RTACOMPLETE
+        )
+        rta_complete_file.touch()
+
+    @staticmethod
+    def create_copy_complete(decrypted_flow_cell: Path, run_dir: Path):
+        """Create a CopyComplete.txt file in the flow cell run directory."""
+        copy_complete_file = Path(
+            run_dir, decrypted_flow_cell.stem, DemultiplexingDirsAndFiles.COPY_COMPLETE
+        )
+        copy_complete_file.touch()
```

Comment:
I guess this will update the timestamp of any existing files. But I guess nothing is dependent on the age of the file?

Reply:
Yep!